pkgs/development/interpreters/lua-5/default.nix

```diff
@@ -22,6 +22,7 @@ in rec {
       # https://github.com/archlinux/svntogit-packages/tree/packages/lua/trunk
       ./liblua.so.patch
     ];
+    postConfigure = ''makeFlagsArray+=(CFLAGS+="-DLUA_USE_READLINE=1" LDFLAGS=-lreadline)'';
```

Comment:
I see there's logic to add readline in `src/Makefile` by using `PLAT=linux-readline` instead of `PLAT=linux`. I wonder if it'd be possible to replace that for lua 5.4?

Reply:
oh , good catch. I have changed that now.